cargo-fix/tests/all/broken_lints.rs

```diff
@@ -0,0 +1,35 @@
+//! Ensure we give good error message when rustfix failes to apply changes
+//!
+//! TODO: Add rustc shim that outputs wrong suggestions instead of depending on
+//! actual rustc bugs!
```

Comment:
https://github.com/rust-lang/rust/issues/50472 will break the test in this file :)